### PR TITLE
Update static.validate.md

### PR DESCRIPTION
**Pull request by pYr0x**
validate function have to return undefined

### CODE REVIEW

File: map/validations/doc/static.validate.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ The error message can be overwritten with `options` __message__ property:
 			 {message: "must be a number"},
 			 function(val){
 					 if( typeof val === 'number' ){
-						 return true
+						 return undefined;
```
Comment (daffl) [inline map/validations/doc/static.validate.md]: I'm not sure if this makes sense. `function(val)` now *always* returns `undefined`.

**Comment by pYr0x** [comment]
http://jsbin.com/zosewigafo/edit?js,console,output

**Comment by pYr0x** [comment]
i will merge that if no one have doubt

cc @justinbmeyer

**Comment by justinbmeyer** [comment]
merge it.

**Pull request merged**